components/LoginModal.vue

```diff
@@ -162,7 +163,13 @@ export default {
     SfLoader,
   },
   setup() {
+    interface loginForm {
```

Comment:
dont define a type here, check existing types, there should be something existing that matches this

Reply:
Updated